### DISCUSSION TITLE
Clean the repository

### DISCUSSION THREAD

**Discussion by fralken** [versat/cntlm#62] [General]
Hi @versat, a quick task could be to keep the repo clean by removing unused or not relevant files.
In my opinion there should not be files related to local ide configuration.
A list of not relevant files could be:

.settings/
scripts/
.cproject
.project
.travis.ymlOLD (I guess this workflow is no longer used)

What do you think?

**Comment by versat** [comment]
- .travis.ymlOLD All actions and even more should already be ported to GitHub actions. So this file should be removed.
- Eclipse project files: While the files are for a local IDE configuration I think it should be possible to adapt it to another environment with only small changes. Since I did not use Eclipse for some time we could remove these files. I am using VS Code mainly at the moment.
What are you using? Should we add some other project files?
- scripts/ I have no idea why this is in the repository. I have not found any reference to it in the repo, so I guess it could be removed without problems. Not sure if someone would be missing it.

**Reply by fralken** [reply]
scripts/ have been added (imo by mistake) in the commit that added pac support. So for sure it can be removed.

I'm using VS Code as well and it is very convenient with the C/C++ extension pack. I excluded the .vscode folder from the project since it can contain local configurations that can be confusing for others.

If you agree all these files can be removed from the repo and the eclipse files (settings/, .cproject, .project) can be added to .gitignore.

**Reply by versat** [reply]
Thanks for looking this up, that makes sense. I agree, then all these files can be removed.

**Reply by fralken** [reply]
Ok I removed the files and added to .gitignore
